src/olympia/reviewers/views.py

```diff
@@ -478,10 +469,6 @@ def _queue(request, TableObj, tab, qs=None, unlisted=False,
         if not is_searching and not admin_reviewer:
             qs = filter_admin_review_for_legacy_queue(qs)
         if not unlisted:
-            qs = filter_static_themes(
-                qs, acl.action_allowed(request, amo.permissions.ADDONS_REVIEW),
-                acl.action_allowed(
-                    request, amo.permissions.STATIC_THEMES_REVIEW))
```

Comment:
A few lines below this we filter the queue with:
```python
        qs = qs.filter(
            Q(addon_type_id=amo.ADDON_STATICTHEME) |
            Q(**{'files.is_webextension': False}) |
            Q(**{'addons_addonreviewerflags.auto_approval_disabled': True})
        )
```

With the `ThemeQueueMixin` you've added we can remove the first of those filters right ?

Reply:
yes.  The rest of the filter will need changing too when search plugins become auto-approved.

Reply:
leaving this as-is because it needs some refactoring and we'll be reworking the queues soon